src/frontend-samples/volume-query-sample/VolumeQueryHelper.tsx

```diff
@@ -0,0 +1,93 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
I think this file should be named after the class it contains: ProgressBar.tsx.

Also, I think this file should be added to the SampleSpec so it is shown in the editor.